appveyor.yml

```diff
@@ -43,13 +44,24 @@ install:
 
 build_script:
   - echo Building on %arch% with %compiler%
+  - set "CFLAGS=-static %CFLAGS%"
+  - set "CXXFLAGS=-static %CXXFLAGS%"
   - cd libvmaf
-  - meson --backend=ninja build
-  - ninja -vC build
+  - meson setup --backend=ninja --default-library=static --buildtype=release --prefix="%CD%/install" build
+  - ninja -vC build install
 
 test_script:
   - ninja -vC build test
 
+after_build:
+  - 7z a vmaf-win-%arch%.zip install\**\*.*
```

Comment:
@1480c1 The zip is now missing vmaf.exe and vmaf_rc.exe (and the corresponding dll's). Shouldn't these be included?

Reply:
I don't think so since those aren't technically installed, so they would not be present in a distribution build (yet)

Reply:
> I don't think so since those aren't technically installed, so they would not be present in a distribution build (yet)

Yes, this is true.